banzai/main.py

```diff
@@ -297,20 +297,23 @@ def run_end_of_night():
             logger.error(logs.format_exception())
```

Comment:
`run_end_of_night` should be removed, as it is not required once we start realtime reductions, and furthermore it is now broken by the removal of `image_type` from `image_utils.select_images`.